test/views/declaration/DeclarantDetailsViewSpec.scala

```diff
@@ -32,7 +32,7 @@ import views.html.declaration.declarant_details
 import views.tags.ViewTest
 
 @ViewTest
-class DeclarantDetailsViewSpec extends UnitViewSpec with ExportsTestData with CommonMessages with Stubs {
+class DeclarantDetailsViewSpec extends UnitViewSpec with ExportsTestData with CommonMessages with Stubs with Injector {
```

Comment:
Should we be testing that the link is visible in the View?

Reply:
You're not using Injector here